jwst/background/background_sub.py

```diff
@@ -81,10 +191,13 @@ def average_background(bkg_list, sigma, maxiters):
         The averaged background exposure
     """
 
-    # Determine the dimensionality of the background files
-    bkg_model = datamodels.open(bkg_list[0])
-    bkg_dim = len(bkg_model.data.shape)
-    image_shape = bkg_model.data.shape[-2:]
+    # Determine the dimensionality of the input file
+    in_model = datamodels.open(input_model)
```

Comment:
If `input_model` is an already open data model, I know it doesn't hurt to call `datamodels.open()` on it again, because that method is smart enough to do nothing when presented with an already open model, but then is it really necessary to be calling `datamodels.open` at all here? Can't `input_model` just be used directly, instead of the new `in_model`?

Reply:
I'd missed this in just translating over the old logic, you're right that input_model is already a data model so this is unnecessary and has been changed